Merge Conflict/Assets/Scripts/UiManager.cs

```diff
@@ -164,6 +164,7 @@ private void SwitchMenu(string requestedMenu)
             case "Settings":
                 AudioManager.Instance.PlayButtonClickSound();
                 OpenMenu(_settings);
+                SettingsMenu.Instance.SetDisplayedSettings();
```

Comment:
naming should follow convention of other Menus and be called SettingsMenu.Instance.OpenMenu()